scripts/load_testing/create_account.py

```diff
@@ -1,14 +1,18 @@
 import os
 
 from faker import Factory
+import foney
```

Comment:
Minor pep8/style quibble. Generally you'd want to import standard libs first, then third-party stuff, then anything internal

Reply:
ahh, that's fine. I can fix that. I'm not super familiar with Py conventions.